components/organisms/TeamSection/TeamSection.tsx

```diff
@@ -0,0 +1,112 @@
+import alexis from "@/public/Alexis.png";
+import shedrack from "@/public/shedrack.png";
+import Link from "next/link";
+
+const teamData = [
+  {
+    userName: "Shedrack Akintayo",
+    position: "Founder",
+    imgUrl: "shedrack.png",
+  },
+  {
+    userName: "Seunbayo Adebayo",
+    position: "Advocacy",
+    imgUrl: "",
+  },
+  {
+    userName: "Amarachi Iheanacho",
+    position: "Technical Writer",
+    imgUrl: "",
+  },
+  {
+    userName: "Damilola Ezekiel",
+    position: "Technical Writer",
+    imgUrl: "",
+  },
+  {
+    userName: "Alexis Abakasanga",
+    position: "Designer",
+    imgUrl: "alexis.png",
+  },
+];
+export default function TeamSection() {
+  function Card({ imgSource, title, position }) {
```

Comment:
instead of having the card component inside the `TeamSection`, add it to the atoms folder and import it here